### PR TITLE
Feature/238 target applicant type

### PR DESCRIPTION
**Pull request by BlancoSebastianEzequiel**

### CODE REVIEW

File: src/graphql/Offer/Mutations/createOffer.ts [hunk 1]
```diff
@@ -15,6 +16,9 @@ export const createOffer = {
     description: {
       type: nonNull(String)
     },
+    target: {
```
Comment (dylanalvarez) [inline src/graphql/Offer/Mutations/createOffer.ts:19]: targetApplicantType ?

File: src/graphql/Offer/Types/GraphQLOffer.ts [hunk 2]
```diff
@@ -37,6 +38,9 @@ const GraphQLOffer = new GraphQLObjectType<Offer, IApolloServerContext>({
     graduadosApprovalStatus: {
       type: nonNull(GraphQLApprovalStatus)
     },
+    target: {
```
Comment (dylanalvarez) [inline src/graphql/Offer/Types/GraphQLOffer.ts:41]: targetApplicantType ?

File: src/models/Offer/Model.ts [hunk 3]
```diff
@@ -77,6 +78,13 @@ export class Offer extends Model<Offer> {
   })
   public graduadosApprovalStatus: ApprovalStatus;
 
+  @Column({
+    allowNull: false,
+    type: ENUM<string>({ values: targetApplicantTypeEnumValues }),
+    ...isTargetApplicantType
+  })
+  public target: TargetApplicantType;
```
Comment (dylanalvarez) [inline src/models/Offer/Model.ts:86]: targetApplicantType ?

File: test/generators/Offer/index.ts [hunk 4]
```diff
@@ -4,13 +4,15 @@ import { OfferRepository } from "$models/Offer";
...
 import { IOfferSection } from "$models/Offer/OfferSection";
+import { TargetApplicantType } from "$models/Offer";
 import { Secretary } from "$models/Admin";
 import { ApprovalStatus } from "$models/ApprovalStatus";
 
 export interface IOfferInput {
   companyUuid: string;
   careers?: IOfferCareer[];
   sections?: IOfferSection[];
+  target?: TargetApplicantType;
```
Comment (dylanalvarez) [inline test/generators/Offer/index.ts:15]: targetApplicantType ?

File: src/seeders/constants/offers/javaSemiSenior.ts [hunk 5]
```diff
@@ -1,10 +1,12 @@
 import { uuids } from "../uuids";
+import { TargetApplicantType } from "../../../models/Offer/Interface";
```
Comment (llmanuel) [inline src/seeders/constants/offers/javaSemiSenior.ts:2]: $models ?

File: src/seeders/constants/offers/javaSenior.ts [hunk 6]
```diff
@@ -1,10 +1,12 @@
 import { uuids } from "../uuids";
+import { TargetApplicantType } from "../../../models/Offer/Interface";
```
Comment (llmanuel) [inline src/seeders/constants/offers/javaSenior.ts:2]: idem

**Pull request merged**